.orchestra/config/global_options.yml

```diff
@@ -22,6 +22,8 @@ gcc_host_version: #@ gcc_host_version
 
 regular_c_compiler: clang
 regular_cxx_compiler: clang++
+win64_c_compiler: x86_64-w64-mingw32-gcc
+win64_cxx_compiler: x86_64-w64-mingw32-g++
```

Comment:
At this stage, I don't see any reason to customize this. Let's inline it where it's used.